client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -178,97 +95,69 @@ const LowerJaw = ({
-              aria-label={t('buttons.get-help')}
-              data-cy='get-help-button'
-              onClick={openHelpModal}
-            >
-              <Help />
-            </button>
-          ) : null}
-        </div>
-      </div>
-    );
-  };
+  const isAttemptsLargerThanTest =
+    currentAttempts &&
+    testsLength &&
+    (currentAttempts >= testsLength || currentAttempts >= 3);
 
   const showDesktopButton = window.innerWidth > MAX_MOBILE_WIDTH;
 
-  const renderButtons = () => {
-    return (
-      <>
-        <div id='action-buttons-container'>
-          {isSignedIn ? null : challengeIsCompleted ? (
-            <Button
-              block={true}
-              href={`${apiLocation}/signin`}
-              className='btn-cta'
-            >
-              {t('learn.sign-in-save')}
-            </Button>
-          ) : null}
-          <button
-            id='test-button'
-            data-cy='run-tests-button'
-            className={`btn-block btn ${challengeIsCompleted ? 'sr-only' : ''}`}
-            aria-hidden={testBtnAriaHidden}
-            onClick={tryToExecuteChallenge}
-          >
-            {showDesktopButton
-              ? t('buttons.check-code')
-              : t('buttons.check-code-2')}
-          </button>
-          <button
-            id='submit-button'
-            data-cy='submit-button'
-            aria-hidden={!challengeIsCompleted}
-            className='btn-block btn'
-            onClick={tryToSubmitChallenge}
-            ref={submitButtonRef}
-          >
-            {t('buttons.submit-and-go')}
-          </button>
-        </div>
-      </>
-    );
-  };
+  const checkButton = showDesktopButton
+    ? t('buttons.check-code')
+    : t('buttons.check-code-2');
+
+  const runningChallenge = challengeIsCompleted && isEditorInFocus;
 
   return (
     <div className='action-row-container'>
-      {renderButtons()}
+      <LowerJawButtons
+        signed={isSignedIn}
+        completeChallenge={challengeIsCompleted}
+        signInText={t('learn.sign-in-save')}
+        excuteChallenge={tryToExecuteChallenge}
+        checkButtonText={checkButton}
+        submitChallenge={tryToSubmitChallenge}
+        submitButtonText={t('buttons.submit-and-go')}
+      />
       <div
         style={runningTests ? { height: `${testFeedbackHeight}px` } : {}}
         className={`test-feedback`}
         id='test-feedback'
         aria-live='assertive'
         ref={testFeedbackRef}
       >
-        {renderTestFeedbackContainer()}
+        {runningTests && (
+          <span className='sr-only'>{t('aria.running-tests')}</span>
+        )}
+        {challengeIsCompleted && (
+          <LowerJawStatus
+            testText={t('learn.test')}
+            showFeedback={isFeedbackHidden}
+            congratulationText={t('learn.congratulations')}
+          >
+            {runningChallenge && (
+              <span className='sr-only'>{t('testScreenReaderText')}</span>
```

Comment:
`testScreenReaderText` doesn't seem to be defined in translations.json. The original sr-only text used here was `aria.submit` ("Use Ctrl + Enter to submit") and was only used when the keyboard focus was still in the editor (the assumption being that the user used Ctrl + Enter to run the tests and thus they would also use Ctrl + Enter to submit their answer). I suppose it wouldn't hurt to show it even if the focus is on the submit button, since you can use use Ctrl + Enter. Or we could remove it completely since screen reader users probably don't need to keep hearing this over and over. But if we did that then it would be nice to document the Ctrl + Enter behavior somewhere so screen reader users will know that it exits.

Reply:
> ("Use Ctrl + Enter to submit") and was only used when the keyboard focus was still in the editor (the assumption being that the user used Ctrl + Enter to run the tests and thus they would also use Ctrl + Enter to submit their answer). 

There is slight confusion on my part, was it originally part of editor? meaning when screen reader focus on the editor, they will hear "Use Ctrl + Enter to submit".

If so, I like that more than the current one, because while navigating to the lower jaw, I was pressing tab multiple times which created tabs in the editor, then I had to press "Ctrl + Enter" to move to the confirm button.

Reply:
> was it originally part of editor? meaning when screen reader focus on the editor, they will hear "Use Ctrl + Enter to submit".

I don't think so. I'm not hearing any such thing in the production version right now. Currently, the only time this text is heard is when the user uses Ctrl + Enter to check their code and it passes. Then we tack on this text to remind them that they can use Ctrl + Enter to submit as well.

> I was pressing tab multiple times which created tabs in the editor

Yes, by default the Tab key is trapped in the editor and you have to use Ctrl + m to "free" it so that you can then Tab to the button below the editor. This is difficult to know about since we don't have any documentation about it.

> Edit: if this isn't the case, maybe we should add screen reader text for keyboard shortcuts' shortcut

This isn't just a screen reader issue, so the solution should not involve sr-only text. The majority of people using this site with the keyboard will not be using a screen reader. So if we need to make sure we have a way to communicate to ALL keyboard users what the shortcuts are. We are getting a little off topic here. I think I might open an issue about this.

Reply:
Ya, this seems out of the scope of the PR 👍 🤔. I think it's better for now to add the original text, and we can replace the text with better communication to the keyboard users. 👍